msg/RadarSector.msg

```diff
@@ -1,4 +1,27 @@
-# A collection of radar scanlines, typically covering a contiguous
+# A collection of marine radar rays, typically covering a contiguous
 # arc of a full circle.
-Header header
-RadarScanline[] scanlines
+
+Header header            # timestamp in the header is the acquisition time of 
+                         # the first ray in the sector.
+                         #
+                         # in frame frame_id, angles are measured around 
+                         # the positive Z axis (counterclockwise, if Z is up)
+                         # with zero angle being forward along the x axis
+
+float32 angle_start      # start angle of this sector [rad]
+float32 angle_increment  # angular distance between measurements [rad]
+
+duration time_increment  # time between rays [seconds] - if your scanner
```

Comment:
In `angle_increment`, you use "measurements" and here "rays". It might be good to unify this terminology.